src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -115,12 +111,39 @@ public InputStream requestStream(
                 Stripe.getApiBase(), e.getMessage()),
             e);
       }
-      handleError(response, apiMode);
+      handleError(response, apiRequest.getApiMode());
     }
 
     return responseStream.body();
   }
 
+  @Override
+  @SuppressWarnings({"TypeParameterUnusedInFormals", "unchecked"})
+  public <T extends StripeObjectInterface> T request(
```

Comment:
Why isn't the default implementation sufficient?

Reply:
Yeah this kind of sucks -- the default implementation actually goes the other way to avoid breaking changes. `StripeResponseGetter.request(ApiRequest req)` is the new one that has the default implementation that delegates to `StripeResponseGetter.request(baseAddress, method, path, params, ...)`